sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -10,24 +10,34 @@
 
 ### Other Changes
 
+## 3.16.4 (2022-05-15)
+
+### Features Added
+
+-Adds support for max integrated cache staleness on item request.
```

Comment:
How about "Adds support for max integrated cache staleness (ref: https://docs.microsoft.com/en-us/azure/cosmos-db/integrated-cache)"